docker/datahub-gms/env/docker.env

```diff
@@ -3,6 +3,7 @@ EBEAN_DATASOURCE_PASSWORD=datahub
 EBEAN_DATASOURCE_HOST=mysql:3306
 EBEAN_DATASOURCE_URL=jdbc:mysql://mysql:3306/datahub?verifyServerCertificate=false&useSSL=true&useUnicode=yes&characterEncoding=UTF-8
 EBEAN_DATASOURCE_DRIVER=com.mysql.jdbc.Driver
+ENABLE_SCSI=false
```

Comment:
We have this flag per entity internally. Do we prefer a global flag here?

Reply:
We should have it per entity here as well. I will make the change